edx_proctoring/templates/proctored_exam/rejected.html

```diff
@@ -8,18 +8,12 @@
 
   <p>
     {% blocktrans %}
-      You are no longer eligible for academic credit for this course, regardless of your final grade.
       If you have questions about the status of your proctored exam results, contact {{ platform_name }} Support.
     {% endblocktrans %}
   </p>
   {% include 'proctored_exam/visit_exam_content.html' %}
 
   <hr>
-  <p>
```

Comment:
I think you should remove the ``<hr>`` too since it is only needed as a separator from this paragraph.